Gateway/Request/CaptureDataBuilder.php

```diff
@@ -204,28 +215,47 @@ protected function getOpenInvoiceData($payment)
     }
 
     /**
-     * Return the data of the multiple capture requests required to capture the full order
+     * Return the data of the multiple capture requests required to capture the full amount OR
+     * multiple capture requests required to capture a partial amount OR
+     * a single capture request required to capture a partial amount
      *
      * @param $payment
      * @param $currency
      * @param $adyenOrderPayments
-     * @return array[]
+     * @param $captureAmount
+     * @return array
      */
-    private function buildMultipleCaptureData($payment, $currency, $adyenOrderPayments)
+    public function buildPartialOrMultipleCaptureData($payment, $currency, $adyenOrderPayments, $captureAmount): array
```

Comment:
Was this made public so that we can cover it with a test?

Reply:
Fixed in https://github.com/Adyen/adyen-magento2/pull/1297/commits/ba4d8fd6c2aa9cc2c508a648d410aae3bc7b7bb7